types/authorization.ts

```diff
@@ -38,16 +38,54 @@ export interface Authorization {
         status: AuthorizationStatus
 
         /**
-         * Optional. The reason the authorization was declined. Available only when status is Declined
+         * Optional. The reason the authorization was declined. Available only when status is Declined.
          */
         declineReason?: string
 
         merchant: Merchant
 
         /**
-         * Indicates whether the authorization is recurring
+         * Indicates whether the authorization is recurring.
          */
         recurring: boolean
+
+        /**
+         * Optional. The payment method used, one of: Manual, Swipe, Contactless, ChipAndPin, Stored, Other.
+         */
+        paymentMethod?: string
+
+        /**
+         * Optional. The type of digital wallet used, one of: Google, Apple, Other.
+         */
+        digitalWallet?: string
+        
+        /**
+         * Optional. The verification method used, one of: Address, CVV2, AddressAndCVV2.
+         */
+        cardVerificationData?: {
+            verificationMethod?: string
```

Comment:
I wonder if `verificationMethod` is actually optional? It would seem to me that if `cardVerificationData` is present, then so should `verificationMethod`. Though, I'm not super sure of this by reading the docs.

Reply:
Yes, it's optional. thanks for this comment